doc/api/n-api.md

```diff
@@ -158,14 +158,14 @@ from version 3 with some additions. This means that it is not necessary
 to recompile for new versions of Node.js which are
 listed as supporting a later version.
 
-|       | 1       | 2        | 3        | 4        |
-|:-----:|:-------:|:--------:|:--------:|:--------:|
-| v6.x  |         |          | v6.14.2* |          |
-| v8.x  | v8.0.0* | v8.10.0* | v8.11.2  | v8.16.0  |
-| v9.x  | v9.0.0* | v9.3.0*  | v9.11.0* |          |
-| v10.x |         |          | v10.0.0  | v10.16.0 |
-| v11.x |         |          | v11.0.0  | v11.8.0  |
-| v12.x |         |          |          | v12.0.0  |
+|       | 1       | 2        | 3        | 4        | 5         |
+|:-----:|:-------:|:--------:|:--------:|:--------:|:---------:|
+| v6.x  |         |          | v6.14.2* |          |           |
+| v8.x  | v8.0.0* | v8.10.0* | v8.11.2  | v8.16.0  | REPLACEME |
```

Comment:
Should these all be REPLACME ?  I know we use that normally for when versions need to inserted but I think that might make sense for only master, which is not listed in the table at all . Maybe we need to add v13.x with the replacement and leave the others blank until the backports are complete?

Reply:
I wasn't sure how to mark them either. Nevertheless, we need to replace the appropriate token on all branches whenever N-API 5 appears in *any* of them.

Reply:
Agreed, its just that I think there is some tooling related to REPLACEME and releases. @BethGriggs might know the specifics off the top of her head.

Reply:
@nodejs/releasers replace all of the `REPLACEME` placeholders when a release is cut: https://github.com/nodejs/node/blob/master/doc/releases.md#step-3-update-any-replaceme-and-dep00xx-tags-in-the-docs

If this landed on master and was then cherry-picked into `12.x` then the entire column would be replaced with the `12.x` release which would not be the desired result.

Reply:
Oh, I see, so, for each backport, the appropriate row must be marked as REPLACEME.